templates/training/index.html

```diff
@@ -452,7 +452,41 @@ <h2 itemprop="name">Charm Developer</h2>
   </div>
 </section>
 
-<section class="p-strip--light">
+<section class="p-strip--light is-bordered" itemscope itemtype="https://schema.org/Product">
+  <div class="row">
+    <div class="col-7">
+      <h2 itemprop="name">Snapcraft 101</h2>
+    </div>
+  </div>
+
+  <div>
+    <div class="u-fixed-width">
+      <hr />
+    </div>
+
+    <div class="row p-divider">
+      <div class="col-7 p-divider__block">
+        <p itemprop="description">A hands-on workshop to learn how to create, build, publish and maintain your own snaps using Snapcraft and deploy them on Ubuntu Core. The workshop alternates short instructor-led discussions with task-focused practical labs. At the end of the two day workshop, your development team will be fully capable of deploying and maintaining your own snaps.</p>
+        <p><a href="https://assets.ubuntu.com/v1/f44d287e-Snapcraft 101 Outline.pdf">Read the course outline (PDF)&nbsp;&rsaquo;</a></p>
+      </div>
+      <div class="col-5 p-divider__block">
+        <dl class="p-inline-definition-list">
+          <dt class="p-inline-definition-list__title">Duration</dt>
+          <dd class="p-inline-definition-list__item">2 days</dd>
+          <dt class="p-inline-definition-list__title">Cost</dt>
+          <dd class="p-inline-definition-list__item" itemprop="offers" itemscope itemtype="https://schema.org/Offer">USD <span itemprop="priceCurrency" content="USD">$</span><span itemprop="price" content="5000">5,000</span> up to 15 attendees</dd>
+          <dt class="p-inline-definition-list__title">Location</dt>
+          <dd class="p-inline-definition-list__item">Remote</dd>
+          <dt class="p-inline-definition-list__title">Availability</dt>
+          <dd class="p-inline-definition-list__item">Private groups only</dd>
+        </dl>
+        <p><a href="/training/contact-us?product=snapcraft-101" data-form-id="1223" data-lp-id="1973" data-return-url="https://www.ubuntu.com/training/thank-you?product=snapcraft-101" data-lp-url="https://www.ubuntu.com/training/thank-you?product=snapcraft-101">Contact us to book yours&nbsp;&rsaquo;</a></p>
```

Comment:
Do you need to formid and returnURL if they are defined on the contact page?

Reply:
I had wondered the same thing, it is a pattern that is used on all of the links so I included it. If you think they can be removed, I can remove them from all of the links on the page?

Reply:
If you can, leaving code in a better way then you found it is always a good thing :+1: thanks.